model/internal/data/protogen/trace/v1/trace.pb.go

```diff
@@ -193,6 +193,9 @@ type ResourceSpans struct {
 	Resource v1.Resource `protobuf:"bytes,1,opt,name=resource,proto3" json:"resource"`
 	// A list of InstrumentationLibrarySpans that originate from a resource.
 	InstrumentationLibrarySpans []*InstrumentationLibrarySpans `protobuf:"bytes,2,rep,name=instrumentation_library_spans,json=instrumentationLibrarySpans,proto3" json:"instrumentation_library_spans,omitempty"`
+
+	// This is expected to be ignored by marshaling/unmarshaling. Double-check that this is true.
+	Context go_opentelemetry_io_collector_model_internal_data.PDataContext
 }
```

Comment:
This needs to be auto-generated

Reply:
That's the part that we don't know how to do nicely. Should we use sed to patch this file as part of `make genproto` target, after Protobuf compiler does its part of the job? Do we have any other options?

Reply:
@bogdandrutu, I'll need your help here.

Reply:
ping